### ISSUE TITLE
Spike: Use an area for Identity?

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#44]
### Expected result

Decide if a dedicated MVC area is better or not.

### Additional context

Add any other context or guidelines here.

### Resources

* https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/areas?view=aspnetcore-3.1

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
